README.md

```diff
@@ -68,6 +73,30 @@ The rate limit is calculated as follows:
 
 This is a linear rate limit that increases over time with a maximum limit. This rate limit is derived from these values which can be set by and admin OR updated by the `CONTROLLER` role. The `CONTROLLER` updates these values to increase/decrease the rate limit based on the functionality within the contract (e.g., decrease the rate limit after minting USDS by the minted amount by decrementing `lastAmount` and setting `lastUpdated` to `block.timestamp`).
 
+## Trust Assumptions
+Below are all stated trust assumptions for using this contract in production:
+- The `DEFAULT_ADMIN_ROLE` is fully trusted, to be run by governance.
+- The `RELAYER` role is assumed to be able to be fully compromised by a malicious actor. 
+  - The logic in the smart contracts must prevent the movement of value anywhere outside of the ALM system of contracts.
+  - The rate limits setup must prevent the possibility of griefing or locking/burning value by a malicious actor.
+  - The `FREEZER` must be able to stop the compromised `RELAYER` from performing more harmful actions by using the `freeze()` function.
+- AAVE deposits are subject to AAVE governance changes, such as setting reserves inactive, paused or frozen. 
+- USDC holdings are subject to blacklist risk.
+- Ethena deposits are subject to USDe depeg risks, blacklist risks, and mint and burn limits.
+- The Ethena Minter is fully trusted. If it is compromised it can: 
+  - Accept orders submitted by a malicious delegated signer that would burn a large amount of USDC for a small amount of USDe (bad quote).
+  - DOS minting/burning of USDe, locking funds.
+- A compromised `RELAYER` can DOS Ethena unstaking, but this can be mitigated by freezing the Controller and reassigning the `RELAYER`. This is outlined in a test `test_compromisedRelayer_lockingFundsInEthenaSilo`.
```

Comment:
I don't think we need to exhaustively list every risk lol. There is an uncountable amount. You've listed USDe having depeg risk, but every derivative has this.

Even just the trust assumptions are incredibly complex for something like Ethena and is best left to risk analysis of onboarding process and not listed in this readme.

Reply:
Agree, I think it would be good to mention that these are examples, so it would not be regarded as a great deal if someone comes up with something that is not listed here.

Reply:
I don't think they are relevant at all for a technical audit. This is the type of analysis that is done by collateral onboarding.

The security model is that when we ship assets to protocol x, we trust whatever assumptions have been made by the collateral onboarding process.

Where the audits should focus is to make sure that whatever the swap mechanism is, there is limited damage the `relayer` can do in terms of slippage and opportunity cost. The trust model assumes that wherever we ship the assets after that the protocol is completely trusted.